src/main/java/com/redhat/cloud/notifications/routers/StatusService.java

```diff
@@ -0,0 +1,39 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.StatusResources;
+import com.redhat.cloud.notifications.models.CurrentStatus;
+import io.smallrye.mutiny.Uni;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import javax.inject.Inject;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+
+import static com.redhat.cloud.notifications.Constants.INTERNAL;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path(INTERNAL + "/status")
+public class StatusService {
+
+    @Inject
+    StatusResources statusResources;
+
+    @GET
+    @Produces(APPLICATION_JSON)
+    @Tag(name = OApiService.PRIVATE)
+    public Uni<CurrentStatus> getCurrentStatus() {
```

Comment:
UI will need to access this endpoint. Currently `/internal` is not accessible unless the proxy is used (which is not the case).

Reply:
Good point, thanks for noticing this.

Reply:
That's fixed.